src/test/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/FhirPayloadTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import org.hl7.fhir.r4.model.HumanName;
+import org.hl7.fhir.r4.model.Patient;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.nhais.parse.FhirParser;
+
+import java.util.Collections;
+
+public class FhirPayloadTest {
```

Comment:
Our future user acceptance tests will cover this, remove?

Reply:
agree I'm removing it.